jaxrs/pom.xml

```diff
@@ -100,8 +98,69 @@
             <artifactId>json-patch</artifactId>
             <version>1.7</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.qmino</groupId>
+            <artifactId>miredot-annotations</artifactId>
+            <version>1.3.1</version>
+            <scope>compile</scope>
+        </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.qmino</groupId>
+                <artifactId>miredot-plugin</artifactId>
+                <version>1.6.1-SNAPSHOT</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>restdoc</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <licence>
+                        cHJvamVjdHxvcmcuamJvc3MuYWVyb2dlYXIudW5pZmllZHB1c2gudW5pZmllZHB1c2gtamF4cnN8MjAxNy0wNS0wMXx0cnVlfC0xI01Dd0NGQitaa0VVRVlsaU9LQjljR2h5TWNSWm1VMTREQWhRbUlHOXY3MExCR0FoT2FubklFSER2SWl1OVZnPT0=
+                    </licence>
+                    <output>
+                        <html>
+                            <baseUrl>https://SERVER:PORT/CONTEXT/rest</baseUrl>
+                            <initialCollapseLevel>100</initialCollapseLevel>
+                            <hideIssuesTab>false</hideIssuesTab>
+                        </html>
+                    </output>
+                    <restModel>
+                        <httpStatusCodes>
+                            <httpStatusCode>
```

Comment:
This is set up default status codes (for example, 200 for all GET methods), so you do not need to write `@statuscode` in javadoc.
I think, that it's nice.